### DISCUSSION TITLE
Img srcset in Safari either squashes the images or not showing at all.

### DISCUSSION THREAD

**Discussion by philipboomy** [livewire/livewire#3126] [7. Bugs (archived)]
### Description

First of all. Thank you for building Livewire (and AlpineJS). Great tools that make life a little easier.

Secondly. This is likely not a bug in Livewire - I cant see why it would be. I am just not sure where to post this issue except here.

I use Livewire with Statamic and when using srcset either vis img or source in picture tag the images are squashed after requesting for example new blog posts with Livewire. On initial load it looks ok.

If I have remove height and width CSS classes the images are not showing at all.

It is happening on a Craft CMS site too which uses HTMX - Similar thing as Livewire. So I guess its a bug in Safari but I wonder if you or anyone else has run into this issue? I am gonna link to this issue when asking HTMX developer if has run into this. Since its happening all Safari browsers iOS and Mac it would be nice to see if anyone has shed some light on this.

Here is a screenshot https://share.getcloudapp.com/nOul86e2 that show the image dimensions are 0x0 after getting a new list of blog post with images using Livewire.

**Comment by joshhanley** [comment]
@philipboomy I'm not exactly sure what you mean by using srcset. Can you provide some working code (as minimal as possible to replicate the issue) that I can run locally to test the issue? Is it something that can be replicated without using Statamic or Craft?

**Comment by philipboomy** [comment]
@joshhanley When using this https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Responsive_images with Livewire then the image will not show. You do not need Statamic. You could use Laravel alone.

Fetch images using Livewire and output them using srcset per Mozilla docs for example and you should be able to see the issue. So not on initial load but afterwards using Livewire. Like filtering for example.

**Comment by joshhanley** [comment]
@philipboomy ah right thanks for the info! 

Any change you could put a small sample repo together that demos the issue, so someone can have a look?

As you say it's probably a Safari bug, but will be good to confirm Livewire's not causing any issues.
